PWGDQ/Tasks/dqEfficiency.cxx

```diff
@@ -837,9 +837,11 @@ struct AnalysisSameEventPairing {
 };
 
 struct AnalysisDileptonTrack {
+  Produces<aod::DQcandidates> dqcandidatesList;
   OutputObj<THashList> fOutputList{"output"};
   // TODO: For now this is only used to determine the position in the filter bit map for the hadron cut
   Configurable<string> fConfigTrackCuts{"cfgLeptonCuts", "", "Comma separated list of barrel track cuts"};
+  Configurable<bool> fConfigFlatTables{"cfgFlatTables", false, "Produce a single flat tables with all relevant information dilepton-track candidates"};
```

Comment:
Maybe name this option to something like "cfgFillCandidateTable" or something like that

Reply:
Done!